File: PPA/Assignment/Input_output/Input_output2/Program5.c
/*Program 5: Write a C Program that accept minutes as input from user and display the
total number of hours and minutes.
Input: 452
Output: 7 Hours, 32 Minutes.  */

#include<stdio.h>

void main() {

	int min;
	int hour;

	printf("Enter the minutes: ");
	scanf("%d",&min);		//take total minutes from user
	
	hour=(int)min/60;		//this gives us the hour
	min=min%60;			//this gives the remaining minutes
	printf("%d Hours, %d Minutes\n",hour,min);
}

File: Java/Assignments/Assignment/CombineAssignment/Program5.java

/*	Program 5:Write a program to print the ASCII value of individual character from a
	string (take hardcoded value)
	
	e.g : abcd
	output : a= 97,b = 98,c=99,d=100
*/


class Ascii {

	public static void main(String[] args) {

		String s = "abcd";

		for(int i=0;i<s.length();i++) {

			char ch =s.charAt(i);
			if(i==s.length()-1)
				System.out.println(ch+"= "+(int)ch+".");
			else
				System.out.print(ch+"= "+(int)ch+",");

		}
			
	}
}
